pmlc/dialect/xsmm/ir/ops.cc

```diff
@@ -85,6 +85,43 @@ ParseResult parseGemmInvokeF32Op(OpAsmParser &parser, OperationState &result) {
       parser.resolveOperands(b.indices, indexType, result.operands));
 }
 
+ParseResult parseBRGemmOffsInvokeF32Op(OpAsmParser &parser,
```

Comment:
Can we use the ODS assembly format instead of hand writing a printer/parser?

See: https://mlir.llvm.org/docs/OpDefinitions/#declarative-assembly-format